src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -779,6 +778,18 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji addEmoji(String name, BufferedImage image){
```

Comment:
@randomname8 no, it has a string header which has to be prepended to the data
@Techtony96 `sx.blah.discord.util.Image`